docs/pages/web/buttontoggle.tsx

```diff
@@ -0,0 +1,317 @@
+
+If ButtonToggle is used as a control button to show/hide a Popover-based component, we recommend passing the following ARIA attributes to assist screen readers:
+- \`accessibilityLabel\`: if present, read by screen readers read instead of the \`text\` prop.
+- \`accessibilityControls\`: informs the screen reader that ButtonToggle controls the display of an anchored Popover-based component. It populates [aria-controls](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html).
+`}
+          title="ARIA attributes"
+        />
+
+        <MainSection.Subsection
+          description={`
+Disabled ButtonToggles do not need to pass color contrast guidelines.
+
+[From w3.org, 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html): Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
+
+Our current disabled ButtonToggle implementation does fail to pass color contrast on accessibility integration tests. To exclude disabled buttontoggles from the integration tests we recomment conditionally setting a \`data-test-id={ isDisabled ? "disabled-button-<name>" : undefined }\` and excluding them from the integration test.
+
+On [cypress-axe](https://www.npmjs.com/package/cypress-axe) that can be achieved with <code>cy.a11yCheck({ exclude: [['[data-test-id="disabled-button-submit"]']] })<code>`}
+          title="Color contrast in disabled state"
+        />
+      </AccessibilitySection>
+
+      <LocalizationSection
+        name={generatedDocGen?.displayName}
+        noDefaultLabelProvider
+        notes="Avoid truncating ButtonToggle text whenever possible. Refer to the [ButtonToggle usage guidelines](#Usage-guidelines) for more information"
+      />
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          description={`ButtonToggle is available in 3 fixed sizes. The ButtonToggle text has always a fixed size of 16px:
+1. \`lg\` (48px)
+    Large is the only size that should be used on Pinner surfaces.
+2. \`md\` (40px)
+    Medium is used on more dense UI such as business surfaces or internal tools.
+3. \`sm\` (32px)
+    Small should be used sparingly and only in places where the UI is very dense.`}
+          title="Size"
+        >
+          {/* @ts-expect-error - TS2322 - Type '{ children: ({ size }: { [key: string]: any; }) => Element; size: string[]; }' is not assignable to type 'IntrinsicAttributes & Props'. */}
+          <CombinationNew size={['sm', 'md', 'lg']}>
+            {({ size }) => (
+              <ButtonToggle
+                accessibilityLabel={`Example size ${size}`}
```

Comment:
can we use the transparent variant here?

Reply:
Done!